g3/git/client.py

```diff
@@ -9,3 +11,24 @@ def commit(message: str) -> None:
     """
     sh = Shell()
     sh.git("commit", "-m", message)
+
+
+def get_commit_messages(origin_branch) -> List[str]:
```

Comment:
Can you add this as an extra `commit_messages` property in `GitInfo` object too as discussed [here](https://github.com/Workable/g3/pull/69/files#r1270540810)?

Reply:
Fixed